### ISSUE TITLE
Automatic updating of outside URL paths for Meteor apps

### ISSUE THREAD

**Issue by mitar** [sandstorm-io/meteor-spk#20]
It seems that it is possible to update [URL from a running app](https://docs.sandstorm.io/en/latest/developing/path/). For Meteor, this could probably be done automatically with some integration with Flow Router.

**Comment by zarvox** [comment]
https://github.com/wekan/wekan/blob/90428e7ba1c0e4185f92dc26b793aff1a9e04bc6/sandstorm.js#L144..L150 is an example of an app doing this.

**Comment by mitar** [comment]
Oho, this is good code. :-) I think this should be some Meteor package. To make it really easy to integrate Meteor with Sandstorm apps. Currently, from what I have seen until now, it looks easy at first, but then there are some edge cases one has to still implement manually.

I will look more into this code and maybe make a package.

**Comment by mitar** [comment]
Yes, this is pretty basic: https://github.com/mitar/council-app/blob/master/packages/council/flow-router/sandstorm.coffee
